packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -209,20 +210,36 @@ export const SideNavLayout = ({ children, groupedRoutes, navOpen: navOpenProp })
   );
 
   const Header = (
-    <PageHeader
-      className="ws-page-header"
-      headerTools={(algolia || hasVersionSwitcher) && <HeaderTools
-        versions={versions}
-        hasSearch={algolia}
-        hasVersionSwitcher={hasVersionSwitcher}
-        hasDarkThemeSwitcher={hasDarkThemeSwitcher}
-        pathPrefix={pathPrefix} />}
-      logo={prnum ? `PR #${prnum}` : <Brand src={logo} alt="Patternfly Logo" />}
-      logoProps={{ href: prurl || pathPrefix || '/' }}
-      showNavToggle
-      topNav={topNavItems.length > 0 && <TopNav navItems={topNavItems} />}
-    />
-  );
+    <Masthead className="ws-masthead">
+      <MastheadToggle>
+        <PageToggleButton variant="plain" aria-label="Global navigation">
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+      <MastheadMain>
+        <MastheadBrand href={prurl || pathPrefix || '/'}>
+          {prnum ? `PR #${prnum}` : (
+            <Brand src={logoBase} alt="PatternFly logo" widths={{ default: '180px', md: '220px' }}>
```

Comment:
oops! I gave you the wrong breakpoint. This will match core.

```suggestion
            <Brand src={logoBase} alt="PatternFly logo" widths={{ default: '180px', '2xl': '220px' }}>
```